packages/store-api/src/typeDefs/index.ts

```diff
@@ -34,6 +38,10 @@ export const typeDefs = [
   ProductGroup,
   Organization,
   AggregateOffer,
+  Mutation,
```

Comment:
This is just a nitpick, but how about moving this `Mutation` type to below `Query`? 😂